ush/python/pygfs/task/aero_analysis.py

```diff
@@ -238,16 +241,29 @@ def get_bkg_dict(self, task_config: Dict[str, Any]) -> Dict[str, List[str]]:
         # Start accumulating list of background files to copy
         bkglist = []
 
-        # aerosol DA needs coupler
-        basename = f'{to_fv3time(task_config.current_cycle)}.coupler.res'
-        bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
-
-        # aerosol DA only needs core/tracer
-        for ftype in ['core', 'tracer']:
-            template = f'{to_fv3time(self.task_config.current_cycle)}.fv_{ftype}.res.tile{{tilenum}}.nc'
-            for itile in range(1, task_config.ntiles + 1):
-                basename = template.format(tilenum=itile)
-                bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+        # if using IAU, we can use FGAT
+        bkgtimes = []
+        if task_config.DOIAU:
+            nowtime = task_config.AERO_WINDOW_BEGIN
+            while nowtime <= add_to_datetime(task_config.current_cycle, to_timedelta(f"{task_config['assim_freq']}H") / 2):
+                bkgtimes.append(nowtime)
+                nowtime = add_to_datetime(nowtime, to_timedelta(f"{task_config['restart_interval']}H"))
```

Comment:
@aerorahul this is the restart_interval spot, like you said, if we do bkg times in __init__, then this is not needed

Reply:
`restart_interval` is a definition in `config.fcst` that controls the interval with which for the (atm) model to dump restarts.  (aerosols are output from the atm component).
It seems the use here is to identify what is the interval between backgrounds. The restarts happen to be the backgrounds for aeroDA.  Would using a different variable to disambiguate between the `restart_interval` that the model defines and the `background_interval` that the DA requires help?
Thoughts?